lib/src/git.rs

```diff
@@ -802,9 +802,16 @@ pub enum FailedRefExportReason {
 
 #[derive(Debug)]
 struct RefsToExport {
-    branches_to_update: BTreeMap<RefName, (Option<gix::ObjectId>, gix::ObjectId)>,
-    branches_to_delete: BTreeMap<RefName, gix::ObjectId>,
-    failed_branches: HashMap<RefName, FailedRefExportReason>,
+    /// Remote bookmark `(ref_name, (old_oid, new_oid))`s to update, sorted by
+    /// `ref_name`.
+    branches_to_update: Vec<(RefName, (Option<gix::ObjectId>, gix::ObjectId))>,
```

Comment:
Perhaps we should add _sorted to the names?

Reply:
I wouldn't because this is internal type, and we would have to rename them back and force when the content becomes sorted/unsorted. For external types (`Vec<FailedRefExport>` and `GitImportStats`), the important property is that lists have stable order.